converter/src/test/java/gov/cms/qpp/conversion/validate/PiSectionValidatorTest.java

```diff
@@ -0,0 +1,79 @@
+package gov.cms.qpp.conversion.validate;
+
+import static com.google.common.truth.Truth.assertThat;
+import static com.google.common.truth.Truth.assertWithMessage;
+
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import gov.cms.qpp.conversion.model.Node;
+import gov.cms.qpp.conversion.model.TemplateId;
+import gov.cms.qpp.conversion.model.error.Detail;
+import gov.cms.qpp.conversion.model.error.ProblemCode;
+import gov.cms.qpp.conversion.model.error.correspondence.DetailsErrorEquals;
+
+class PiSectionValidatorTest {
+
+	private static final String VALID_ACI_MEASURE = "ACI_EP_1";
```

Comment:
Does this check for the 2017 ACI_EP_1 measure only or does it need to switch between PI_EP_1?

Reply:
This is an old test that I was refactoring but it uses old test data. Good catch, as the actual measure name is just test data here but it should be updated as well.